### PR TITLE
DRAFT: Merge tool qa2

### PR DESCRIPTION
**Pull request by mitch-mitchel**
create for ease of talking about code

### CODE REVIEW

**Comment by janebitovi** [comment]
lg2m

File: CAJA_WS.php [hunk 1]
```diff
@@ -166,17 +166,160 @@
...
+		{
+			$filename=GUIDES_DIR.$row['filename'];
+			$path_parts = pathinfo($filename);
+			$filedir = $path_parts['dirname'];
+			$ownerfolder = substr($row['filename'], 0, stripos($row['filename'], "/", 0));
+
+			$allfiles = scandir($filedir);
+			$mediaFiles = array_filter($allfiles, is_media_file);
+			$templateFiles = array_filter($allfiles, is_template_file);
+			// template1.json, template1.pdf
```
Comment (mitch-mitchel) [inline CAJA_WS.php]: comments here can probably be removed

File: CAJA_WS.php [hunk 2]
```diff
@@ -166,17 +166,160 @@
...
+			$sourcePaths = [];
+			while ($path = $res->fetch_assoc()) {
+				$sourcePaths[$path['gid']] = pathinfo(GUIDES_DIR.$path['filename'])['dirname'];
+			}
+
+			foreach ($files as $file) {
+				$filename = $file['filename'];
+				$rename = $file['rename'] ?: $filename;
+				$extension = $file['extension'];
+				// $sourceGid = $file['sourceGid'];
```
Comment (mitch-mitchel) [inline CAJA_WS.php]: also these 2 can be removed

File: package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "@caliorg/a2jdeps": "^7.1.3",
-    "@caliorg/a2jviewer": "^8.0.0",
+    "@caliorg/a2jviewer": "file:../viewer",
```
Comment (mitch-mitchel) [inline package.json]: set this back to whatever the latest viewer is

**Review by mitch-mitchel — COMMENTED** [review]
just some comments cleanup - left over from me :-)

**Pull request merged**